### ISSUE TITLE
Command application output wrong.

### ISSUE THREAD

**Issue by RockerHX** [JohnSundell/Files#61]
Code:
```
import Foundation
import Files

print(Folder.current.path)
print(Folder.home.path)
```
Xcode `run` or `marathon run` output is right.
```
/Users/xxx/Desktop/AppIcon/
/Users/xxx/
```

When I use command application: `exec`, output result is wrong.
```
/Users/xxx/
/Users/xxx/
```

**Comment by clayellis** [comment]
@RockerHX Pardon the potentially obvious question but, are you running `exec` from your home directory?

**Comment by RockerHX** [comment]
@clayellis No, I try run it on different folders. I don't know other people have encountered the same problem, or my own system or environment problems?
```
macOS High Sierra
10.13.4 (17E202)

Apple Swift version 4.1 (swiftlang-902.0.48 clang-902.0.37.1)
Target: x86_64-apple-darwin17.5.0
```

**Comment by Samasaur1** [comment]
This is correct. When running an executable from the command line, the current directory is the directory the executable is run from. However, if an executable is double-clicked, the current directory is always the home directory.

I don't have an Apple reference handy, but I tested it on my laptop:

**Setup for tests:**
```bash
$ cd Desktop
$ echo "pwd" > workingDir
$ chmod +x workingDir
```
**Run tests from terminal:**
```bash
$ pwd
/Users/Samasaur1/Desktop
$ cat workingDir
pwd
$ ./workingDir
/Users/Samasaur1/Desktop
$ mkdir newDir
$ cd newDir
$ pwd
/Users/Samasaur1/Desktop/newDir
$ ../workingDir
/Users/Samasaur1/Desktop/newDir
$ cd ../../..
$ pwd
/Users
$ ./Samasaur1/Desktop/workingDir
/Users
```
**Output from double-clicking:**
```bash
$ /Users/Samasaur1/Desktop/test ; exit;
/Users/Samasaur1
logout
Saving session...
...copying shared history...
...saving history...truncating history files...
...completed.

[Process completed]
```

Also, I am aware that Gradle, a Java build tool, says this:

> by default we should be in the correct project dir, but when run from Finder on Mac, the cwd is wrong

That's from Gradle-generated executables fro Java command-line projects.



In summary, this is Apple's fault (or intent). To avoid this, don't double-click your executable. You could try and use a wrapper script that simply calls your script, or some other method, but Apple has ensured that double-slicking will not give you the output you are expecting. This isn't a problem with Files.
